helpers/cron-requester/lib/index.js

```diff
@@ -0,0 +1,110 @@
+const BandChain = require('@bandprotocol/bandchain.js')
+const { CronJob } = require('cron')
+const fs = require('fs')
+
+// Due to how bandchain.js is written, we cannot batch all the requests in one tx yet
+// TODO: Fix this
+async function runJob(bandchain, mnemonic, validatorCounts, requests) {
+  let count = 0
+  for (let request of requests) {
+    try {
+      const requestId = await bandchain.submitRequestTx(
+        request.oracleScript,
+        request.params,
+        validatorCounts,
+        mnemonic,
+      )
+
+      count++
+      console.log(
+        '∟ ✅ requestId = %s | oracleScript #%d %s',
+        requestId,
+        request.oracleScriptId,
+        JSON.stringify(request.params),
+      )
+    } catch {
+      console.log(
+        '∟ ⛔️ request failed | oracleScript #%d %s',
+        request.oracleScriptId,
+        JSON.stringify(request.params),
+      )
+    }
+  }
+  console.log(
+    '%s [%d/%d] requests went through',
+    count === requests.length ? '⛳️' : '👎',
+    count,
+    requests.length,
+  )
+  console.log('--------------------------------------------------------')
+}
+
+async function start(configFilePath) {
+  let config = {}
+
+  try {
+    // Load config file
+    const configFile = fs.readFileSync(configFilePath)
+    config = JSON.parse(configFile)
+  } catch {
+    throw new Error('Incorrect config file path/format')
+  }
+
+  const {
+    mnemonic,
+    chainId,
+    endpoint,
+    cronPattern,
+    validatorCounts,
+    requests,
+  } = config
+
+  // Check format
```

Comment:
Change comment to "Check request parameter variable types" might make it clearer?